pkg/xds/generator/admin_proxy_generator.go

```diff
@@ -3,14 +3,45 @@ package generator
 import (
 	core_xds "github.com/kumahq/kuma/pkg/core/xds"
 	xds_context "github.com/kumahq/kuma/pkg/xds/context"
+	envoy_common "github.com/kumahq/kuma/pkg/xds/envoy"
 	envoy_clusters "github.com/kumahq/kuma/pkg/xds/envoy/clusters"
 	envoy_listeners "github.com/kumahq/kuma/pkg/xds/envoy/listeners"
 	envoy_names "github.com/kumahq/kuma/pkg/xds/envoy/names"
 )
 
 // OriginAdmin is a marker to indicate by which ProxyGenerator resources were generated.
 const OriginAdmin = "admin"
-const adminReadyEndpoint = "/ready"
+
+var staticEnpointPaths = []*envoy_common.StaticEndpointPath{
+	{
+		Path:        "/clusters",
+		RewritePath: "/clusters",
+	},
+	{
+		Path:        "/config_dump",
+		RewritePath: "/config_dump",
+	},
+	{
+		Path:        "/listeners",
+		RewritePath: "/listeners",
+	},
+	{
+		Path:        "/listeners",
+		RewritePath: "/listeners",
+	},
+	{
+		Path:        "/quitquitquit",
+		RewritePath: "/quitquitquit",
+	},
+	{
+		Path:        "/ready",
+		RewritePath: "/ready",
+	},
+	{
+		Path:        "/stats",
+		RewritePath: "/stats",
+	},
+}
```

Comment:
what do you think about just exposing all endpoints (prefix `/`)?

in routes section we can first add a route for `/ready` without auth and then add the rest of them with auth

Reply:
That makes sense of course, I was not sure if we want all or just a subset.
Then we need to make sure the order is correct.